backend/server.ts

```
@@ -0,0 +1,366 @@
+        name: { $regex: new RegExp(`^${projectName}$`, "i") }
+      });
+      
+      // Extract metadata from the project document
+      let modelMetadata = {
+        filename: `${projectName}.ifc`,
+        element_count: elements.length,
+        upload_timestamp: new Date().toISOString(),
+        project_id: null as string | null
+      };
+      
+      if (project && project.metadata) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)